frontend/src/component/common/FeatureStatusChip/FeatureStatusChip.tsx

```diff
@@ -19,7 +21,7 @@ export const FeatureStatusChip = ({
     const value = stale ? 'Stale' : 'Active';
 
     return (
-        <div data-loading style={{ marginLeft: '8px' }}>
+        <div data-loading style={{ marginLeft: withLeftMargin ? '8px' : 0 }}>
```

Comment:
You can change 8px -> spacing(1)

`<Box sx={theme => ({ marginLeft: theme.spacing(1) })}>`